afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/AbstractScript.java

```diff
@@ -50,6 +52,36 @@ private void processEvents(List<NodeEvent> events, String nodeId, List<ScriptLis
         }
     }
 
+    public void addGenericScript(GenericScript genericScript) {
+        orderedDependencyManager.appendDependencies(INCLUDED_SCRIPTS_DEPENDENCY_NAME, Collections.singletonList(genericScript));
+    }
+
+    public void addScript(T includeScript) {
+        orderedDependencyManager.appendDependencies(INCLUDED_SCRIPTS_DEPENDENCY_NAME, Collections.singletonList(includeScript));
+    }
+
+    public void removeScript(String scriptNodeId) {
+        orderedDependencyManager.removeDependencies(INCLUDED_SCRIPTS_DEPENDENCY_NAME, Collections.singletonList(scriptNodeId));
+    }
+
+    @Override
+    public String readScript(boolean withIncludes) {
+        String ownContent = readScript();
+        if (withIncludes) {
+            String includesScript = orderedDependencyManager
+                    .getDependencies(INCLUDED_SCRIPTS_DEPENDENCY_NAME, AbstractScript.class)
+                    .stream()
+                    // TODO(mathbagu) add commented header for script ? (script project node name ?)
```

Comment:
Strange to assign me a TODO.

Reply:
you told me to last time we talk about that unassigned todo

Reply:
though it was a question, I then decided there would not be a commented header for included scripts